test/src/test/java/org/springframework/security/test/web/configurers/UrlAuthorizationConfigurerAnonymousTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2002-2016 the original author or authors.
```

Comment:
The test should probably go in the existing test at [UrlAuthorizationConfigurerTests](https://github.com/spring-projects/spring-security/blob/802f3186a73c1980d87736537f78b5df10a376b7/config/src/test/java/org/springframework/security/config/annotation/web/configurers/UrlAuthorizationConfigurerTests.java)

Reply:
Yes, that seems like the best place. I removed the new file, and added the test to `UrlAuthorizationConfigurerTests`.